common/containers/Home/index.jsx

```diff
@@ -23,14 +24,22 @@ export class Home extends Component {
     }
   }
 
+  handleNavigateUsers() {
+    if (__CLIENT__) {
+      window.location.href = '/users'
+    }
+  }
+
   render() {
+    const isAdmin = this.props.authData.roles.includes('admin')
```

Comment:
Heading in the right direction, but let's make this more easily accessible throughout the app by setting this state in the `auth` redux store.